packages/jaeger-ui/src/components/TracePage/KeyboardShortcutsHelp.js

```diff
@@ -49,6 +49,10 @@ const descriptions = {
   zoomInFast: 'Zoom in — Large',
   zoomOut: 'Zoom out',
   zoomOutFast: 'Zoom out — Large',
+  collapseAll: 'Collapse All',
+  expandAll: 'Expand All',
+  collapseOne: 'Collapse One',
```

Comment:
Maybe this should be `Collapse One Level` and `Expand One Level` to make it clearer in the help modal?